omniauth-dice.gemspec

```diff
@@ -32,7 +32,7 @@ client certificates.'
   spec.add_development_dependency 'rack'
   spec.add_development_dependency 'rack-test'
   spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop', '~> 0.41.1'
```

Comment:
This will need to be a hard-lock, version `0.41.2` of Rubocop is what required Ruby 2.0 support so a pessimistic lock on the patch will still pull in the breaking version.